aiida/__init__.py

```diff
@@ -34,10 +34,12 @@
 __version__ = '1.5.0'
 __authors__ = 'The AiiDA team.'
 __paper__ = (
-    'G. Pizzi, A. Cepellotti, R. Sabatini, N. Marzari, and B. Kozinsky,'
-    '"AiiDA: automated interactive infrastructure and database for computational science", '
-    'Comp. Mat. Sci 111, 218-230 (2016); https://doi.org/10.1016/j.commatsci.2015.09.013 '
-    '- http://www.aiida.net.'
+    'Sebastiaan. P. Huber, Spyros Zoupanos, Martin Uhrin, Leopold Talirz, Leonid Kahle, Rico Häuselmann, '
+    'Dominik Gresch, Tiziano Müller, Aliaksandr V. Yakutovich, Casper W. Andersen, Francisco F. Ramirez, Carl S. Adorf,'
+    ' Fernando Gargiulo, Snehal Kumbhar, Elsa Passaro, Conrad Johnston, Andrius Merkys, Andrea Cepellotti, '
+    'Nicolas Mounet, Nicola Marzari, Boris Kozinsky, and Giovanni Pizzi, *AiiDA 1.0, a scalable computational '
+    'infrastructure for automated reproducible workflows and data provenance*, Scientific Data **7**, 300 (2020); '
+    'DOI: [10.1038/s41597-020-00638-4](https://doi.org/10.1038/s41597-020-00638-4)'
```

Comment:
For this string I would keep a text-only format, rather than markdown, so just put the DOI URL. Also, I would put shortened names (this will be printed in a few places) just with initials - or even simply S.P.Huber et al.? But at least use the initials, to shorten a bit.

Finally, this string is printed also in headers of files. I hope that in 2020 Unicode is supported, but one cannot be so sure (e.g. this is printed in the header of xmgrace files, that most probably could get confused) - so maybe it's safer use the less-nice form `Häuselmann` -> `Haeuselmann` and `Müller` -> `Mueller`

```suggestion
    'https://doi.org/10.1038/s41597-020-00638-4'
```